samples/matter/lock/sample.yaml

```
@@ -107,6 +107,18 @@ tests:
     tags:
       - sysbuild
       - ci_samples_matter
+  sample.matter.lock.schedule:
```

Maybe we could name it the same as snippet:
```suggestion
  sample.matter.lock.schedules:
```